spark/v3.3/build.gradle

```diff
@@ -31,7 +31,9 @@ configure(sparkProjects) {
   configurations {
     all {
       resolutionStrategy {
+        force "com.fasterxml.jackson.module:jackson-module-scala_${scalaVersion}:2.13.4"
```

Comment:
jackson-module-paranamer isn't required to be forced here because it doesn't appear to be used in Spark 3.3